### DISCUSSION TITLE
what is the max  cluster connections can submariner support?

### DISCUSSION THREAD

**Discussion by pengbinbin1** [submariner-io/submariner-operator#2203] [Q&A] [answered]

**Comment by skitt** [comment]
@mkolesnik could you take care of this one?

**Comment by mkolesnik** [comment, marked as answer]
Generally it depends on the capabilities of your different clusters and the underlying hardware.

As of recently, we performed limited scale testing on a very specific scenario:
* A single machine hosted all clusters.
* All clusters used [KIND](https://github.com/kubernetes-sigs/kind).
* All clusters had 1 node only (control plane).
* Not using globalnet.
* Not using Lighthouse.

In these tests the maximum scale was ~30 clusters before Submariner became unstable.
As it was very limited scenario, the actual scale cap can be quite higher when using actual clusters.
